src/utils_db_query.c

```diff
@@ -265,7 +265,7 @@ static int udb_result_submit (udb_result_t *r, /* {{{ */
     }
     else
     {
-      char tmp[DATA_MAX_NAME_LEN];
+      char tmp[DATA_MAX_NAME_LEN - 10];
```

Comment:
Why 10?

Reply:
No good reason. The error was "output 2 or more bytes (assuming 129) into a destination of size 128" in line 280 below, so anything over 2 should work. Want me to change it to 4 to match the other adjustment?

Reply:
it looks like DATA_MAX_NAME_LEN is defined to be 128. Is tmp[] here related to the ident_str in the other file? Would it make sense to leave this as DATA_MAX_NAME_LEN, but change the other one to use DATA_MAX_NAME_LEN too?

Reply:
No, they are unrelated. I can make the adjustment amount consistent, though...